### PR TITLE
V0.16.1

### PR DESCRIPTION
**Pull request by Blanca-Esqueda**
Description
"Replace" button modal not triggering correctly for elements beside the first one.
The inline_entity_form and extra patch didn't fix the modal.
This seems to be related to entity_browser issue. There are issues on D.O related to wrong triggering for the remove button, and some other for the replace button but there is not one for this issue is specific.
If instead of modal an iframe is used for the entity_browser, then everything works.

Related Tickets
https://kanopi.teamwork.com/#/tasks/25619060

Observations:
It includes the configuration changes for add the new entity browser for the iframe and use that one in the outcome goals section.
configuration files were placed in their correspondent folder, and uuid and _core info removed.

### CODE REVIEW

File: profiles/rescue_x_base/config/install/entity_browser.browser.image_entity_browser_iframe.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+uuid: c07d0932-347d-40ab-b9fc-eb0e51923e41
```
Comment (thejimbirch) [inline profiles/rescue_x_base/config/install/entity_browser.browser.image_entity_browser_iframe.yml]: The UUID needs to be removed here. @Blanca-Esqueda

File: modules/other/ef_comms_common/config/install/user.role.fundraiser.yml [hunk 2]
```diff
@@ -4,22 +4,49 @@ dependencies:
...
+  - 'edit call_to_action_7 embeddable content'
+  - 'edit campaigns'
+  - 'edit simple_sticky_footer embeddable content'
   - 'update content translations'
   - 'use text format basic_html'
+  - 'view campaigns'
   - 'view embeddable content in test mode'
+  - 'view test campaign content'
   - 'view the administration theme'
-third_party_settings:
```
Comment (thejimbirch) [inline modules/other/ef_comms_common/config/install/user.role.fundraiser.yml]: Probably should not be removing this.

File: modules/other/ef_comms_common/config/install/user.role.fundraiser.yml [hunk 3]
```diff
@@ -4,22 +4,49 @@ dependencies:
...
 weight: 4
 is_admin: null
 permissions:
+  - 'access campaign overview'
   - 'access content overview'
   - 'access embeddable content overview'
   - 'access image_entity_browser entity browser pages'
+  - 'access image_entity_browser_iframe entity browser pages'
   - 'access toolbar'
-  - 'create media'
```
Comment (thejimbirch) [inline modules/other/ef_comms_common/config/install/user.role.fundraiser.yml]: Probably should not be removing this.

File: modules/other/ef_comms_common/config/install/user.role.advocate.yml [hunk 4]
```diff
@@ -4,22 +4,31 @@ dependencies:
...
+  - 'delete campaigns'
   - 'delete content translations'
+  - 'edit campaigns'
   - 'update content translations'
   - 'use text format basic_html'
+  - 'view campaigns'
   - 'view embeddable content in test mode'
+  - 'view test campaign content'
   - 'view the administration theme'
-third_party_settings:
```
Comment (thejimbirch) [inline modules/other/ef_comms_common/config/install/user.role.advocate.yml]: Probably should not be removing this.

File: modules/other/ef_comms_common/config/install/user.role.advocate.yml [hunk 5]
```diff
@@ -4,22 +4,31 @@ dependencies:
...
 weight: 5
 is_admin: null
 permissions:
+  - 'access campaign overview'
   - 'access content overview'
   - 'access embeddable content overview'
   - 'access image_entity_browser entity browser pages'
+  - 'access image_entity_browser_iframe entity browser pages'
   - 'access toolbar'
-  - 'create media'
```
Comment (thejimbirch) [inline modules/other/ef_comms_common/config/install/user.role.advocate.yml]: Probably should not be removing this.

File: modules/other/ef_comms_common/config/install/user.role.fundraiser.yml (hunk 2 above)
Comment (Blanca-Esqueda) [reply modules/other/ef_comms_common/config/install/user.role.fundraiser.yml]: it is not removed,  it just changed to like 7 on the configuration file.

File: modules/other/ef_comms_common/config/install/user.role.advocate.yml (hunk 4 above)
Comment (Blanca-Esqueda) [reply modules/other/ef_comms_common/config/install/user.role.advocate.yml]: it is not removed, it just changed to line 9 on the configuration file. @thejimbirch

File: modules/other/ef_comms_common/config/install/user.role.advocate.yml (hunk 5 above)
Comment (Blanca-Esqueda) [reply modules/other/ef_comms_common/config/install/user.role.advocate.yml]: the new configuration permissions are coming from the dev db:
Beside putting back 'create media', should I remove the following?
 'access campaign overview'
'add campaigns'

Comment (thejimbirch) [reply modules/other/ef_comms_common/config/install/user.role.advocate.yml]: @rbomhof Can you advise?

Comment (rbomhof) [reply modules/other/ef_comms_common/config/install/user.role.advocate.yml]: Not clear to me how the label would have changed with this config, or how media creation would have changed. The Advocate role should have the "access campaign overview" permission, as well as "create media" and "access campaign overview"

File: modules/other/ef_comms_common/config/install/user.role.fundraiser.yml (hunk 3 above)
Comment (rbomhof) [reply modules/other/ef_comms_common/config/install/user.role.fundraiser.yml]: Fundraiser should also have access to "create media", along with "access campaign overview"

File: modules/other/ef_comms_common/config/install/user.role.advocate.yml (hunk 5 above)
Comment (Blanca-Esqueda) [reply modules/other/ef_comms_common/config/install/user.role.advocate.yml]: Put back "create media",  and left the extra role permissions.

File: modules/other/ef_comms_common/config/install/user.role.fundraiser.yml (hunk 3 above)
Comment (Blanca-Esqueda) [reply modules/other/ef_comms_common/config/install/user.role.fundraiser.yml]: Added "create media", and left the extra permissions too.

File: profiles/rescue_x_base/config/install/entity_browser.browser.image_entity_browser_iframe.yml (hunk 1 above)
Comment (Blanca-Esqueda) [reply profiles/rescue_x_base/config/install/entity_browser.browser.image_entity_browser_iframe.yml]: UUID removed

**Pull request merged**